RedHatInsights/compliance-frontend
# feat: RHICOMPL-2124 - Show group categories for rules

This introduces tree table support for the underlying utility hooks for tables and makes use of it in the rules tables in compliance. 

How to test:

1) Append `ruleGroups=enable` to any page/url in compliance
Test a rules table on:

2) a Policy details page
3) a Systems details page
3.1) a Systems details page with multiple policies (my stage accound has such a system)
4) a Report details page
5) when editing a policy
6) in the "Create Policy" wizard.

Please do verify also with the feature flag disabled (`ruleGroups=disable`).



## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
